docs/sources/operators-guide/monitoring-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -0,0 +1,335 @@
+              regex: ""
+              replacement: k8s-cluster
+              separator: ""
+              source_labels:
+                - cluster
+              target_label: cluster
+          scheme: https
+          tls_config:
+            ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+            insecure_skip_verify: false
+            server_name: kubernetes
+        - job_name: integrations/grafana-mimir/metrics
+          kubernetes_sd_configs:
+            - role: pod
+          relabel_configs:
+            - action: keep
+              regex: .*metrics
+              source_labels:
+                - __meta_kubernetes_pod_container_port_name
+            - action: keep
+              regex: mimir-distributed-.*
+              source_labels:
+                - __meta_kubernetes_pod_label_helm_sh_chart
+            - action: replace
+              regex: ""
+              replacement: k8s-cluster
+              separator: ""
+              source_labels:
+                - cluster
+              target_label: cluster
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_namespace
+              target_label: namespace
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_pod_name
+              target_label: pod
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_pod_container_name
+              target_label: container
+            - action: replace
+              separator: ""
+              source_labels:
+                - __meta_kubernetes_pod_label_name
+                - __meta_kubernetes_pod_label_app_kubernetes_io_component
+              target_label: __tmp_component_name
+            - action: replace
+              separator: /
+              source_labels:
+                - __meta_kubernetes_namespace
+                - __tmp_component_name
+              target_label: job
+            - action: replace
+              source_labels:
+                - __meta_kubernetes_pod_node_name
+              target_label: instance
+  global:
+    scrape_interval: 15s
+  wal_directory: /tmp/grafana-agent-wal
+server:
+  http_listen_port: 12345
+```
+
+## Collect metrics and logs without the Helm chart
+
+You can still use the dashboards and rules in the monitoring-mixin even if Mimir or GEM is not deployed via the Helm
+chart or if you are using the deprecated enterprise-metrics Helm chart for GEM.
+As a starting point, use the Agent configuration
+from [Collect metrics and logs via Grafana Agent](#collect-metrics-and-logs-via-grafana-agent) as a starting point.
+There may be some
+modifications you need to make to it. See [dashboards and alerts requirements]({{< relref "requirements.md" >}}) for
+more information.
+
+### Service discovery
+
+The Agent configuration relies on Kubernetes service discovery and pod labels to constrain the collected metrics and
```

Comment:
Is this "pod" or "pods’"

Reply:
"pod labels" = "labels of pods"

I think it is "pod", not "pods"

Reply:
I am likely over thinking this. Keep as is.